spyder/plugins/editor/plugin.py

```diff
@@ -2293,14 +2299,17 @@ def update_cursorpos_actions(self):
         self.next_cursor_action.setEnabled(self.cursor_pos_index is not None \
                     and self.cursor_pos_index < len(self.cursor_pos_history)-1)
 
-    def add_cursor_position_to_history(self, filename, position, fc=False):
+    def add_cursor_position_to_history(self, filename, position, line, column,

```

Comment:
Do we still need to use the cursor `position` in this method? I think with `line` and `column` should be enough but I'm not really sure.

Reply:
It doesn't cost us to have the position, and some of the methods that call this function have a calculation of the position before sending it

Reply:
Ok, no problem then.